stdlib/sys.mli

```diff
@@ -354,6 +354,19 @@ val runtime_warnings_enabled: unit -> bool
 
     @since 4.03.0 *)
 
+val enable_float_array_alloc_warning: int option -> unit
+(** Control whether the OCaml runtime system can emit warnings when values
```

Comment:
```suggestion
(** Control whether the OCaml runtime system emits warnings when values
```

Reply:
I was copying the above `enable_runtime_warnings` but this proposition looks better!